src/ocean_emulators/config.py

```diff
@@ -197,37 +216,171 @@ class BlockConfig(BaseConfig):
     upscale_factor: int = 4
     norm: NormType = "batch"
 
+    def build(self) -> CoreBlockBuilder:
+        match self.activation:
+            case "relu":
+                activation: type[nn.Module] = ReLU
+            case "capped_gelu":
+                activation = CappedGELU
+            case _:
+                raise ValueError(f"Activation {self.activation!r} not supported.")
```

Comment:
```suggestion
               assert_never(self.activation)
```
This will cause a type error if you forget a case. (Will also need to import from typing)

Reply:
(same for the below matches)

Reply:
The assert never stmt caught an error that I missed!! :)